api/pkg/app/app.go

```diff
@@ -218,3 +240,14 @@ func initLogger(mode EnvMode) (*zap.SugaredLogger, error) {
 	}
 	return log.Sugar(), nil
 }
+
+// configFileURL will look for CONFIG_FILE_URL to be defined among
+// environment variables
+func configFileURL() (string, error) {
+
+	if val := viper.GetString("CONFIG_FILE_URL"); val != "" {
+		return val, nil
```

Comment:
could change this to 
```
val = viper.GetString ..
if  val == "" {
   return "", error 
}
return val, nil
```
so that the exception is inside the if